diff --git a/dist/index.d.ts b/dist/index.d.ts
index v0.34.4..v0.34.4
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -115,6 +115,6 @@
-        <Expected>(...MISMATCH: MismatchArgs<Equal<Actual, Expected, Options['branded']>, Options['positive']>): true;
+        <Expected>(...MISMATCH: MismatchArgs<Equal<Actual, Expected>, Options['positive']>): true;
         <Expected>(expected: Expected, ...MISMATCH: MismatchArgs<Equal<Actual, Expected, Options['branded']>, Options['positive']>): true;
     };
     toBeCallableWith: Options['positive'] extends true ? (...args: Params<Actual>) => true : never;
     toBeConstructibleWith: Options['positive'] extends true ? (...args: ConstructorParams<Actual>) => true : never;
-    toHaveProperty: <K extends string>(key: K, ...MISMATCH: MismatchArgs<Extends<K, keyof Actual>, Options['positive']>) => K extends keyof Actual ? ExpectTypeOf<Actual[K], Options> : true;
+    toHaveProperty: <K extends PropertyKey>(key?: K, ...MISMATCH: MismatchArgs<Extends<K, keyof Actual>, Options['positive']>) => K extends keyof Actual ? ExpectTypeOf<Actual[K], Options> : true;
diff --git a/dist/vendor-node.b14cb9ad.js b/dist/vendor-node.b14cb9ad.js
index v0.34.4..v0.34.4
--- a/dist/vendor-node.b14cb9ad.js
+++ b/dist/vendor-node.b14cb9ad.js
@@ -11958,3 +11958,2 @@
-    const suite = lastSuite;
-    while (lastSuite !== file && lastSuite.end < index)
-      lastSuite = suite.suite;
+    while (lastSuite !== file && lastSuite.end < index)
+      lastSuite = lastSuite.suite;
