diff --git a/./dist/vendor-cli-api.f9adf98c.js b/./dist/vendor-cli-api.f9adf98c.js
index v0.32.2..v0.32.2
--- a/./dist/vendor-cli-api.f9adf98c.js
+++ b/./dist/vendor-cli-api.f9adf98c.js
@@ -11958,3 +11958,2 @@
-    const suite = lastSuite;
-    while (lastSuite !== file && lastSuite.end < index)
-      lastSuite = suite.suite;
+    while (lastSuite !== file && lastSuite.end < index)
+      lastSuite = lastSuite.suite;
